documentation/tech-papers/lbft-reconfig/lbft-reconfig.tex

```diff
@@ -36,8 +36,8 @@
 \input{setup}
 
 \begin{document}
-\title{Reconfiguration of the Diem Blockchain by Self-Reflecting Transactions}
```

Comment:
This may very well should be removed

Reply:
@zekun000 ?

Reply:
why?

Reply:
no point in keeping papers for something that no longer exists

Reply:
what do you mean no longer exists? reconfiguration is still there? this still describes how it works

Reply:
Eee, my bad, I felt I had read a different title. Indeed, this is still very relevant.